Task 5/app.js

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import passport from 'passport';
+
+import {productsRouter} from './routes/productsRouter.js';
+import {usersRouter} from './routes/usersRouter.js';
+import {authRouter} from './routes/authRouter.js';
+
+export const app = express();
+
+
+app.use(passport.initialize());
+//passport.use
+//choose one
+import {} from './middlewares/passportLocalMiddleware.js'
```

Comment:
What are you trying to do here?
This is considered redundant since you're regestering your middleware directly to passport instance when you do "passport.use()"